.github/workflows/release.yml

```diff
@@ -1,16 +1,37 @@
-name: release
+name: 'New Release'
+run-name: 'Release ${{ inputs.version_number }}'
+
 on:
-  push:
-    tags:
-      - 'v*'
-jobs: 
-  goreleaser:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: 'Version number (e.g., v1.0.0, v1.0.0-pre, v1.0.0-pre1)'
```

Comment:
should we add a step to validate that the provided tag is correct? One common error is to forget the V

Reply:
yes this should be simple to include and has value